package.json

```diff
@@ -22,18 +24,32 @@
   "license": "MIT",
   "dependencies": {
     "baconjs": "^0.7.83",
```

Comment:
all of these are now `devDependencies`. also, remember to remove jquery here after rewriting all methods using vanilla js